controllers/get-controllers.js

```diff
@@ -29,9 +30,15 @@ exports.getArticle = (req, res, next) => {
 };
 
 exports.getArticles = (req, res, next) => {
-  selectArticles().then((articles) => {
-    res.status(200).send({ articles });
-  });
+  const { sort_by } = req.query;
+  const { order } = req.query;
```

Comment:
You can combine these two lines of code, you can destructure two properties at once!